src/providerhub/azext_providerhub/generated/_params.py

```diff
@@ -0,0 +1,347 @@
+                                                                 'Failed', 'Succeeded', 'MovingResources',
+                                                                 'TransientFailure', 'RolloutInProgress']), help='')
+        c.argument('subscription_state_override_actions', action=AddSubscriptionStateOverrideActions, nargs='+',
+                   help='', arg_group='Subscription Lifecycle Notification Specifications')
+        c.argument('soft_delete_ttl', help='', arg_group='Subscription Lifecycle Notification Specifications')
+        c.argument('provider_hub_metadata_provider_authorizations',
+                   action=AddProviderHubMetadataProviderAuthorizations, nargs='+', help='', arg_group='Provider Hub '
+                   'Metadata')
+        c.argument('resource_provider_authentication', action=AddProviderAuthentication, nargs='+', help='',
+                   arg_group='Provider Hub Metadata')
+        c.argument('authorizations', action=AddAuthorizations, nargs='+', help='', arg_group='Provider Hub Metadata '
+                   'Third Party Provider Authorization')
+        c.argument('managed_by_tenant_id', type=str, help='', arg_group='Provider Hub Metadata Third Party Provider '
+                   'Authorization')
+
+    with self.argument_context('providerhub provider-registration update') as c:
+        c.argument('provider_namespace', type=str, help='The name of the resource provider hosted within ProviderHub.',
+                   id_part='name')
+        c.argument('provider_authentication', action=AddProviderAuthentication, nargs='+', help='')
+        c.argument('provider_authorizations', action=AddProviderAuthorizations, nargs='+', help='')
+        c.argument('namespace', type=str, help='')
+        c.argument('provider_version', type=str, help='')
+        c.argument('provider_type', arg_type=get_enum_type(['NotSpecified', 'Internal', 'External', 'Hidden',
+                                                            'RegistrationFree', 'LegacyRegistrationRequired',
+                                                            'TenantOnly', 'AuthorizationFree']), help='')
+        c.argument('required_features', nargs='+', help='')
+        c.argument('capabilities', action=AddCapabilities, nargs='+', help='')
+        c.argument('metadata', type=validate_file_or_dict, help='Any object Expected value: json-string/@json-file.')
+        c.argument('template_deployment_options', action=AddTemplateDeploymentOptions, nargs='+', help='')
+        c.argument('schema_owners', nargs='+', help='', arg_group='Management')
+        c.argument('manifest_owners', nargs='+', help='', arg_group='Management')
+        c.argument('incident_routing_service', type=str, help='', arg_group='Management')
+        c.argument('incident_routing_team', type=str, help='', arg_group='Management')
+        c.argument('incident_contact_email', type=str, help='', arg_group='Management')
```

Comment:
I see a lot of empty help messages for the arguments. Please add the right messages for them.